tests/chart_tests/test_houston_configmap.py

```diff
@@ -80,47 +80,6 @@ def test_houston_configmap_with_azure_enabled():
     assert livenessProbe["periodSeconds"] == 10
 
 
-def test_houston_configmap_with_kubed_enabled():
```

Comment:
See my previous comment about the mounted volume if configSyncer is enabled.

I think we should keep these tests cases but check for astronomer.configSyncer.enabled instead of kubed.enabled

Reply:
Yes, I'll put this back if we opt to go with the `.Values.astronomer.configSyncer.enabled` path you proposed.